Web/View/LandingPages/Edit.hs

```diff
@@ -57,20 +59,17 @@ orderAndRenderParagraphs ctas quotes =
         quotes' = quotes
             |> fmap (\paragraph -> (paragraph.weight, paragraphTitleAndOps EditParagraphQuoteAction paragraph.id paragraph.title ("Quote" :: Text)))
 
+
         allSorted = ctas' ++ quotes'
             |> sortOn fst
             |> fmap snd
 
+        -- Show the paragraph title and the operations to perform on it.
         paragraphTitleAndOps action id title type_  =
```

Comment:
I'm assuming I need to generlize the type signature, and help the compiler. I'm trying to tell it something along

```haskell
paragraphTitleAndOps :: (PrimaryKey (GetTableName record), HasPath controller) => (Id' record -> controller) -> Id' record -> Text -> Text -> Html
paragraphTitleAndOps action id title type_  =
```

But I'm getting it wrong..

Reply:
`paragraphTitleAndOps :: (Show (PrimaryKey record), HasPath controller) => (Id' record -> controller) -> (Id' record -> controller) -> Id' record -> Text -> Text -> Html` works